go/teams/transactions.go

```diff
@@ -313,7 +329,14 @@ func (tx *AddMemberTx) addMemberByUPKV2(ctx context.Context, user keybase1.UserP
 	// No going back after this point!
 
 	tx.sweepKeybaseInvites(uv.Uid)
-	tx.sweepCryptoMembers(uv.Uid)
+
+	// An admin is only allowed to remove an owner UV when, in the same link, replacing them with
+	// a 'newer' UV with a greater eldest seqno.
+	// So, if we're an admin re-adding an owner who does not yet have a PUK
+	// then don't try to remove the owner's pre-reset UV.
+	exceptAdminsRemovingOwners := !hasPUK
+	tx.team.G().Log.CDebugf(ctx, "xxx exceptAdminsRemovingOwners: %v", exceptAdminsRemovingOwners)
```

Comment:
are we keeping `xxx` or should it go?

Reply:
thanks! If you ever see xxx in my PRs it's accidental